vendor_blueprints/verify_refresh_status.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import sys
+import pydomo
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_FINAL_STATUS_SUCCESS = 0
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_ACCOUNT = 201
+EXIT_CODE_DATASET_NOT_FOUND = 202
+EXIT_CODE_EXECUTION_ID_NOT_FOUND = 203
+EXIT_CODE_FINAL_STATUS_INVALID = 204
+EXIT_CODE_FINAL_STATUS_CANCELLED = 205
+EXIT_CODE_STATUS_INCOMPLETE = 206
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--secret-key', dest='secret_key', required=True)
+    parser.add_argument('--dataset-id', dest='dataset_id', required=True)
```

Comment:
Since every other Blueprint now requires username/password, can we see about using that here instead of the client_id/secret_key? Want to keep the authorization consistent across the Blueprints.